locale/en.yml

```diff
@@ -735,13 +735,13 @@ en:
       LdapServer:               LDAP Server
       LoadBalancer:             Load Balancer
       ManageIQ::Providers::BaseManager:                                     Provider
-      ManageIQ::Providers::AnsibleTower::ConfigurationManager:              Configuration Manager (Ansible Tower)
-      ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript: Job Template (Ansible Tower)
+      ManageIQ::Providers::AnsibleTower::AutomationManager:                 Automation Manager (Ansible Tower)
+      ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript: Job Template (Ansible Tower)
```

Comment:
@mzazrivec Should these kinds of changes stay in this PR, or would you prefer they were generated by you separately?